entgql/schema.go

```diff
@@ -727,6 +727,9 @@ func (e *schemaGenerator) mapScalar(gqlType string, f *gen.Field, ant *Annotatio
 			// will generate an enum named "TodoStatus".
 			scalar = gqlType + scalar
 		}
+		if strings.HasPrefix(f.Type.String(), "[]") {
```

Comment:
Use `RType.Kind` instead.

Also, can you please add test cases for this in `internal/todo`? Thanks

Reply:
Sorry.
RType is nil.

Reply:
It's `nil` only for fields without GoType/JSON configured. Use: `f.Type.RType != nil && f.Type.RType.Kind = reflect.Slice`.

Reply:
ok

Reply:
I'm not sure if there is a case when `Kind` is `reflect.Array`, but can we add it as a backup?

Reply:
ok